src/avt/Database/Database/avtGenericDatabase.C

```diff
@@ -3070,21 +3070,6 @@ avtGenericDatabase::GetMesh(const char *meshname, int ts, int domain,
             return NULL;
         }
 
-        //
-        // VTK creates a trivial producer for each data set.  It later does
-        // garbage collection and that takes a long time if we have a lot
-        // of trivial producers.  Make one trivial producer for all
-        // data sets here.
-        //
-        static vtkTrivialProducer* tp = NULL;
-        if(tp == NULL)
-        {
-            tp = vtkTrivialProducer::New();
-            vtkVisItUtility::RegisterStaticVTKObject(tp);
-        }
-        tp->SetOutput(mesh);
-        tp->SetOutput(NULL);
-
         AssociateBounds(mesh);
```

Comment:
Our comments for this section you pulled out specifically state it was there for performance reasons.
Are you implying by its removal the the performance hit no longer exists?

Reply:
The commit message mentions that it doesn't work with the new pipeline API. @cjh1, do you remember anything about this?

Reply:
Sorry no

Reply:
I'll take this commit out and see what breaks in ParaView.